### PR TITLE
Update content for Sketch plugin with Create App story

### PR DESCRIPTION
**Pull request by hanastasov**
Closes https://github.com/IgniteUI/indigo-design-docfx/issues/571

### CODE REVIEW

File: en/sync-themes-plugin.md [hunk 1]
```diff
@@ -6,7 +6,15 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
...
+The Indigo.Design Sketch Plugin allows to create and edit themes over your designs, and quickly create and publish apps and prototypes in Indigo.Cloud from these designs.
+
+Once a design in Sketch is created with the Indigo.Design System, use the plugin to:
+
+-  quickly create an app in Indigo.Design AppBuilder. This app follows the structure of the Sketch design, mapping any symbol from the Indigo.Design UI kit to the corresponding component from Ignite UI. The plugin also takes the images used in the Sketch and upload them in the Assets library under AppBuilder, benefiting from [`Assets` support](https://www.infragistics.com/community/blogs/b/jason_beres/posts/indigo-design-app-builder-october-release-with-assets-support).
```
Comment (StefanIvanov) [inline en/sync-themes-plugin.md:13]:
```suggestion
-  quickly create an app in Indigo.Design AppBuilder. This app follows the structure of the Sketch design, mapping any symbol from the Indigo.Design UI kit to the corresponding component from Ignite UI. The plugin also takes the images used in Sketch and uploads them in the Assets library under AppBuilder, benefiting from [`Assets` support](https://www.infragistics.com/community/blogs/b/jason_beres/posts/indigo-design-app-builder-october-release-with-assets-support).  
```

File: en/sync-themes-plugin.md [hunk 2]
```diff
@@ -81,7 +89,14 @@ When you are ready to publish your work as a prototype to the Indigo.Design Clou
...
 
-### 1. Publishing new prototype
+### 1. Create an application in AppBuilder
+Take your designs and get interactive, responsive app with real UI components and styling. Use the `Create App` tab and the plugin will transform your Sketch design into a real application under the AppBuilder, allowing access to the images and themes that you have used in Sketch - all of this directly in the cloud.
+
+The plugin allows you to choose a name and resolution for your app, as well as choosing your Indigo.Cloud workspace, where to have the app created.
+
+<img class="responsive-img" src="images/create-from-sketch.gif" srcset="images/create-from-sketch.gif 2x" />
+
+### 2. Publishing new prototype
```
Comment (StefanIvanov) [inline en/sync-themes-plugin.md:101]:
```suggestion
### Publishing new prototype
```

File: en/sync-themes-plugin.md [hunk 3]
```diff
@@ -97,7 +112,7 @@ Once you click the Publish button, the project is uploaded to the Indigo.Design
 
 <img class="responsive-img" src="images/Sync_themes_plugin_New_prototype.png" srcset="images/Sync_themes_plugin_New_prototype@2x.png 2x" />
 
-### 2. Update an existing prototype
+### 3. Update an existing prototype
```
Comment (StefanIvanov) [inline en/sync-themes-plugin.md:117]:
```suggestion
### Update an existing prototype
```

File: en/sync-themes-plugin.md [hunk 4]
```diff
@@ -18,7 +26,9 @@ The Indigo.Design Plugin is available for download from [Indigo.Design Cloud](ht
 <img class="responsive-img" src="images/indigo-design-resources.gif" />
 <p style="text-align:center;">Indigo.Design resources</p>
 
-## 2. Themes
+## 2.
```
Comment (StefanIvanov) [inline en/sync-themes-plugin.md:29]:
```suggestion
## 2. Create an application in AppBuilder
Take your designs and get interactive, responsive app with real UI components and styling. Use the `Create App` tab and the plugin will transform your Sketch design into a real application under the AppBuilder, allowing access to the images and themes that you have used in Sketch - all of this directly in the cloud.

The plugin allows you to choose a name and resolution for your app, as well as choosing your Indigo.Cloud workspace, where to have the app created.

<img class="responsive-img" src="images/create-from-sketch.gif" srcset="images/create-from-sketch.gif 2x" />
```

**Review by StefanIvanov — CHANGES_REQUESTED** [review]
I moved a few things around and feel that it is better this way. Aside from that the content you added looks great. Do you think we should also update the rest of the images with the new looks and design?

Thanks! Updating the rest of the images should happen once the plugin is there with the new screens, so let's postpone it for that moment.

**Comment by hanastasov** [comment]
> I moved a few things around and feel that it is better this way. Aside from that the content you added looks great. Do you think we should also update the rest of the images with the new looks and design?

Thanks! Updating the rest of the images should happen once the plugin is there with the new screens, so let's postpone it for that moment.

**Review by StefanIvanov — APPROVED** [review]

**Comment by hanastasov** [comment]
Closing in favour of #573

**Review by StefanIvanov — APPROVED** [review]

**Review by zdrawku — APPROVED** [review]

**Pull request merged**